cluster/manager/manager.go

```diff
@@ -1132,6 +1227,14 @@ func (c *ClusterManager) initListeners(
 			return ErrNewNodeOverMaxCapacity(clusterMaxSize)
 		}
 
+		if len(c.selfClusterDomain) > 0 {
+			// Add an entry of our cluster domain if it does not exists
```

Comment:
this can race with someone else setting the domain to inactive ? don't think we need it here ..

Reply:
after our yesterday's conversation, do you think this is still true?

Reply:
👍